packages/devtools_app/test/association_variable_test.dart

```diff
@@ -0,0 +1,184 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+import 'package:vm_service/vm_service.dart';
+
+import 'package:devtools_app/src/debugger/debugger_controller.dart';
+import 'package:devtools_app/src/debugger/debugger_model.dart';
+import 'package:devtools_app/src/globals.dart';
+import 'package:devtools_app/src/service_manager.dart';
+import 'package:devtools_app/src/vm_service_wrapper.dart';
+
+void main() {
+  ServiceConnectionManager manager;
```

Comment:
can you use the FakeServiceManager class we use elsewhere in DevTools tests?

Reply:
Done, though I hit some issues with the isolate manager which I worked around by updating DebuggerController to avoid switchToIsolate logic for testing.